packages/tangle/storage.go

```diff
@@ -464,26 +464,48 @@ func (s *Storage) Prune() error {
 	return nil
 }
 
+// DBStatsResult is a structure containing all the statistics retrieved by DBStats() method.
+type DBStatsResult struct {
+	StoredCount                   int
+	SolidCount                    int
+	BookedCount                   int
+	ScheduledCount                int
+	SumSolidificationReceivedTime time.Duration
+	SumBookedReceivedTime         time.Duration
+	SumSchedulerReceivedTime      time.Duration
+	SumSchedulerBookedTime        time.Duration
+	MissingMessageCount           int
+}
+
 // DBStats returns the number of solid messages and total number of messages in the database (messageMetadataStorage,
 // that should contain the messages as messageStorage), the number of messages in missingMessageStorage, furthermore
 // the average time it takes to solidify messages.
-func (s *Storage) DBStats() (solidCount, messageCount int, sumSolidificationTime int64, missingMessageCount int) {
+func (s *Storage) DBStats() (res DBStatsResult) {
```

Comment:
:+1: 
Way better than returning lots of vars

Reply:
I would only maybe move the function and `DBStats` results to the `metrics` module (and pass `Storage` as parameter) to keep `storage.go` clean of any metrics logic since it should only be focused on its businnes logic

since it was this way before this is optional imo

Reply:
Would be nice, but this method needs access to private fields, which would be impossible in the metrics module. So I'll leave as it is.